tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -18,7 +18,10 @@
   mat-icon-button
   [matMenuTriggerFor]="menu"
   aria-label="Display notification messages"
+  class="notification-icon-button"
```

Comment:
consideration for future:  you may want to make this button disabled while the data is being fetched.

Reply:
Noted. Thanks!